array.go

```diff
@@ -1092,3 +1093,39 @@ func (a *Array) DeleteFragments(startTimestamp, endTimestamp uint64) error {
 
 	return nil
 }
+
+// DeleteFragments deletes the range of fragments from startTimestamp to endTimestamp
+func DeleteFragments(tdbCtx *Context, uri string, startTimestamp, endTimestamp uint64) error {
+	curi := C.CString(uri)
+	defer C.free(unsafe.Pointer(curi))
+
+	ret := C.tiledb_array_delete_fragments_v2(tdbCtx.tiledbContext, curi,
+		C.uint64_t(startTimestamp), C.uint64_t(endTimestamp))
+	if ret != C.TILEDB_OK {
+		return fmt.Errorf("Error deleting fragments from array: %s", tdbCtx.LastError())
+	}
+
+	return nil
+}
+
+// DeleteFragmentsList deletes the fragments of the list
+func DeleteFragmentsList(tdbCtx *Context, uri string, fragmentURIs []string) error {
+	curi := C.CString(uri)
+	defer C.free(unsafe.Pointer(curi))
+
+	var list []*C.char
+	for _, furi := range fragmentURIs {
+		cfuri := C.CString(furi)
+		defer C.free(unsafe.Pointer(cfuri))
+		list = append(list, cfuri)
+	}
+
+	ret := C.tiledb_array_delete_fragments_list(tdbCtx.tiledbContext, curi, (**C.char)(unsafe.Pointer(&list[0])), C.size_t(len(list)))
```

Comment:
This is probably something we should handle more broadly in the future, but you can use `unsafe.SliceData` to get the address of the start of a slice.

Reply:
Yes that was the initial code but needed to revert because there are some go1.19 installations and 1.19 does not have `SliceData`